pom.xml

```
@@ -55,27 +55,68 @@
             <artifactId>hibernate-core</artifactId>
             <version>5.6.8.Final</version>
         </dependency>
+        <dependency>
+            <groupId>com.auth0</groupId>
+            <artifactId>java-jwt</artifactId>
+            <version>3.10.3</version>
+        </dependency>
+        <dependency>
+            <groupId>io.github.cdimascio</groupId>
+            <artifactId>java-dotenv</artifactId>
+            <version>5.2.2</version>
+        </dependency>
```

```suggestion
        <dependency>
            <groupId>io.github.cdimascio</groupId>
            <artifactId>dotenv-java</artifactId>
            <version>2.2.3</version>
        </dependency>
```
Please use this dependency instead according to their [documentation](https://github.com/cdimascio/java-dotenv#why)

---

Sure!